geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreFactory.scala

```diff
@@ -183,6 +193,10 @@ object KafkaDataStoreFactory {
     val CacheConsistency = new GeoMesaParam[Duration]("kafka.cache.consistency", "Check the feature cache for consistency at this interval", deprecatedParams = Seq(DeprecatedConsistency))
     val CacheTicker      = new GeoMesaParam[Ticker]("kafka.cache.ticker", "Ticker to use for expiring/cleaning feature cache")
     val CqEngineCache    = new GeoMesaParam[java.lang.Boolean]("kafka.cache.cqengine", "Use CQEngine-based implementation of live feature cache", default = false, deprecatedKeys = Seq("useCQCache"))
+    val EventTime        = new GeoMesaParam[java.lang.Boolean]("kafka.cache.eventtime", "Use Event Time for managing cache of features", default = false)
+    val EventTimeAttribute = new GeoMesaParam[String]("kafka.cache.eventtime.attribute", "Use the configured attribute to compute Event Time", default = "")
+    val EventTimeOrdering  = new GeoMesaParam[java.lang.Boolean]("kafka.cache.eventtime.ordering", "Use order of event time to update a feature or not", default = false)
+    val EventTimeExpiry    = new GeoMesaParam[java.lang.Boolean]("kafka.cache.eventtime.expiry", "Use event time to determine expiry of features", default = false)
```

Comment:
can we change the keys to `event-time`? that's the pattern we've been following

Reply:
got it.